airgun/views/computeresource.py

```diff
@@ -122,27 +127,43 @@ def is_displayed(self):
 class ResourceProviderDetailView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
     compprofiles = Text("//a[text()='Compute profiles']")
+    flavor = FilteredDropdown(id='s2id_compute_attribute_vm_attrs_flavor_id')
+    availability_zone = FilteredDropdown(
+        id='s2id_compute_attribute_vm_attrs_availability_zone')
+    subnet = FilteredDropdown(id='s2id_compute_attribute_vm_attrs_subnet_id')
+    security_groups = MultiSelect(
+        id='ms-compute_attribute_vm_attrs_security_group_ids')
+    managed_ip = FilteredDropdown(
+        id='s2id_compute_attribute_vm_attrs_managed_ip')
+    submit = Text('//input[@name="commit"]')
 
     @property
     def is_displayed(self):
         breadcrumb_loaded = self.browser.wait_for_element(
             self.breadcrumb, exception=False)
         return (
-                breadcrumb_loaded
-                and self.breadcrumb.locations[0] == 'Compute resources'
-                and self.browser.wait_for_element(
-                    self.compprofiles, exception=False) is not None
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Compute resources'
+            and self.browser.wait_for_element(
```

Comment:
Next piece of code should suffice our needs
```
        return (
            breadcrumb_loaded
            and self.breadcrumb.locations[0] == 'Compute resources'
            and self.breadcrumb.read() != Create Compute Resource'
        )
```
then logic in:
https://github.com/SatelliteQE/airgun/blob/master/airgun/navigation.py#L73

will verify whether we opened valid entity